src/main/frontend/multi-pipeline-graph-view/app.tsx

```diff
@@ -5,11 +5,15 @@ import { MultiPipelineGraph } from "./multi-pipeline-graph/main";
 
 import "./app.scss";
 import "./multi-pipeline-graph/styles/main.scss";
+import { I18NProvider } from "../common/i18n/i18n-provider";
+import { ResourceBundleName } from "../common/i18n/translations";
 
 const App: FunctionComponent = () => {
   return (
     <div>
-      <MultiPipelineGraph />
+      <I18NProvider bundles={[ResourceBundleName.messages]}>
+        <MultiPipelineGraph />
```

Comment:
Just had a thought, you might need to update how the `I18NProvider` gets the locale, it might just be updating the jelly but it might also be a little more involved (maybe lifting locale evaluation outside the `I18NProvider` and passing it as a prop)

Reply:
done in 803d0fc